### PR TITLE
Terra-fying Terra!

### LINKED ISSUE #665: Design a new logo
When the new name for Saturn is released, design a new logo to represent the application.

### PR DESCRIPTION
**Pull request by elizzhao**
Updates to landing page per mocks
Change logo and beta tag
"Explore tools" button points to internal "Code & Tools" tab from Library 

Fixes #665

### CODE REVIEW

File: src/icons/logo-grey.svg [hunk 1]
```diff
@@ -1,22 +1,9 @@
...
-    <path fill="#fff" d="M283.4 101.3c-.6-1.2-1.2-2.3-1.8-3.5-.9 2.3-1.9 4.7-3 7.1-5.4 11.9-13.9 24.5-26.9 36.8L237 156.4c-.6 3.9-1.5 8.3-2.9 12.8-4.7 15.2-14.7 32.4-32.5 43.5-10.6 6.6-23.9 11-40.7 11.6h-.5c-8.6 0-15.6-7-15.6-15.6 0-1.8 0-3.8.2-6l-51.4 45.2-30.3 26.7c2.9 2.3 5.8 4.6 8.8 6.7 23.2 16.2 51.4 25.6 81.9 25.6 41 0 78-17.2 104.2-44.8 9.4-9.9 17.4-21.1 23.6-33.4 10-19.6 15.7-41.8 15.7-65.3 0-22.2-5.1-43.3-14.1-62.1z"/>
-  </g>
+<svg xmlns="http://www.w3.org/2000/svg" id="Layer_1" viewBox="0 0 460.4 500">
```
Comment (zarsky-broad) [inline src/icons/logo-grey.svg]: Remove id plz

File: src/icons/logo.svg [hunk 2]
```diff
@@ -1,16 +1,9 @@
...
-  <path d="M98 90.3C91.7 66 80.8 43 65.7 22.8 46.2 35.5 29.9 52.7 18.5 73c19.6 18 42.7 31.7 67.8 40.5 2.5-8.4 6.4-16.2 11.7-23.2zm3.7 17.6c-1.4 3-2.6 6.1-3.6 9.2 1.7.5 3.3.9 5 1.3-.4-3.5-.9-7-1.4-10.5z" fill="#546ea7"/>
-  <path fill="#8a4f65" d="M209.2 16c-.9-.5-1.8-.9-2.6-1.4-.2.8-.4 1.7-.6 2.6 1-.5 2.1-.9 3.2-1.2zm63.5 65.2c-.6-1.2-1.2-2.3-1.8-3.5-.9 2.3-1.9 4.7-3 7.1 1.6-1.1 3.3-2.3 4.8-3.6zm-68.9-48.3l-.6.2c-1 7.4-1.5 14.9-1.6 22.4 10.8-5.6 24.2-1.4 29.8 9.4 5.6 10.8 1.4 24.2-9.4 29.8-5.2 2.7-11.3 3.2-16.9 1.5 1.2 6.4 2.8 12.7 4.6 18.8 11.8-3.8 23.3-8.7 34.2-14.6a93.72 93.72 0 0 0 18.5-37.2C252 48 238.9 34.8 223.6 24.5c-7 1.6-13.9 4.2-20.3 7.5 0 .2 0 .4-.1.5l.6.4z"/>
-  <path fill="#8a4f65" d="M203.3 32.6c0 .2-.1.3-.1.5l.6-.2-.5-.3z"/>
-  <path fill="#c55a68" d="M272.7 81.2c-1.5 1.2-3.1 2.4-4.7 3.6-5.4 11.9-13.9 24.5-26.9 36.8l-14.7 14.7c-.6 4.3-1.6 8.6-2.9 12.8 11.9 22.8 28.1 43 47.7 59.5 20.4-39.8 20.9-86.9 1.5-127.4zm-57.8 49.2l17.6-17.5c4.1-3.9 7.9-8 11.4-12.4-10.9 6-22.4 10.9-34.2 14.6 1.5 5.2 3.3 10.3 5.2 15.3z"/>
+<svg xmlns="http://www.w3.org/2000/svg" id="Layer_1" viewBox="0 0 145 160.6">
```
Comment (zarsky-broad) [inline src/icons/logo.svg]: Remove id plz

**Review by MattPutnam — APPROVED** [review]
👍

File: src/pages/LandingPage.js [hunk 3]
```diff
@@ -44,12 +44,16 @@ const LandingPage = pure(() => {
       }
     }, [
       img({ src: landingPageHero, style: { position: 'absolute', right: 0, top: 60 } }),
-      div({ style: { fontSize: 22 } }, 'Welcome to'),
-      div({ style: { fontSize: 85 } }, 'Terra'),
+      div({ style: { fontSize: 45, color: '#359448' } }, 'Welcome to Terra.'),
+      div({ style: { fontSize: 24 } }, 'Terra is a cloud-native platform for'),
```
Comment (zarsky-broad) [inline src/pages/LandingPage.js]: Maybe combine these divs into one with `whiteSpace: 'pre-line'`?

File: src/components/TopBar.js [hunk 4]
```diff
@@ -71,8 +71,8 @@ const styles = {
 const betaTag = b({
   style: {
     fontSize: 8, lineHeight: '9px',
-    color: 'white', backgroundColor: colors.brick,
-    padding: '3px 5px', marginLeft: '0.4rem', verticalAlign: 'middle',
+    color: 'white', backgroundColor: '#73AD43',
```
Comment (zarsky-broad) [inline src/components/TopBar.js:74]: Is this color a one-off?

**Review by zarsky-broad — APPROVED** [review]
Terra-fic.

File: src/components/TopBar.js (hunk 4 above)
Comment (elizzhao) [reply src/components/TopBar.js:74]: Probably not, but we don't have the updated style guide yet so I figured to do it this way until then.

File: src/icons/logo-grey.svg [hunk 5]
```diff
@@ -1,22 +1,9 @@
...
+  <style>
+    .st0{fill:none;stroke:#fff;stroke-width:4.2591;stroke-miterlimit:10}.st1{fill:#fff}
```
Comment (panentheos) [inline src/icons/logo-grey.svg]: This class name already exists in a couple other icons. I don't think they're actually used, but generally we should still be careful about these names.

**Review by panentheos — APPROVED** [review]
Looks great!

**Review by katevoss — APPROVED** [review]
Looks good! One note that I brought up with Liz is that the Sign In page says Terra and has the logo. But Lou is working on redesigning the signin page so it will be resolved soon.
screen shot 2018-12-05 at 11 40 57 am

**Pull request merged**